cmd/rpc.go

```diff
@@ -73,50 +67,3 @@ func inprocSend(rpcCmd common.RpcCmd, requestData interface{}, responseData inte
 	}
 	return nil
 }
-
-////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-// NewHttpClient returns the instance of struct containing an instance of http.client and url
```

Comment:
These methoda are unused? why we are removing this as part of linting fixes;

Reply:
unused